main.go

```diff
@@ -173,8 +175,12 @@ func parseFlags() *config {
 	cfg.promlogConfig.Format = &promlog.AllowedFormat{}
 	a.Flag("log.format", "Output format of log messages. One of: [logfmt, json]").
 		Envar("PROMBQ_LOG_FORMAT").Default("logfmt").SetValue(cfg.promlogConfig.Format)
-
+	googleProjectIDFlagCause := a.Flag("googleProjectID", "The GCP Project ID is mandatory when googleAPIjsonkeypath is not provided").Envar("PROMBQ_GCP_PROJECT_ID")
 	_, err := a.Parse(os.Args[1:])
+	if cfg.googleAPIjsonkeypath == "" {
+		googleProjectIDFlagCause.Required().StringVar(&cfg.googleProjectID)
+		_, err = a.Parse(os.Args[1:])
+	}
```

Comment:
This isn't working as intended. I get a `Error parsing commandline arguments: required flag --googleProjectID not provided` when I try to run it with `--googleAPIjsonkeypath`.

Reply:
see above